_posts/0000-01-02-NishaSathish.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Man! I used to hate coding once upon a time. Never thought that I would end up here